Decisions.Equifax/EquifaxUtilities.cs

```diff
@@ -0,0 +1,159 @@
+using Decisions.Equifax.ConsumerCreditReport.Request;
+using Decisions.Equifax.ConsumerCreditReport.Response;
+using Decisions.Equifax.PreQualificationOfOne.Response;
+using DecisionsFramework;
+using DecisionsFramework.Design.ConfigurationStorage.Attributes;
+using DecisionsFramework.Design.Flow;
+using DecisionsFramework.Design.Flow.CoreSteps;
+using DecisionsFramework.Design.Flow.Mapping;
+using DecisionsFramework.Design.Properties.Attributes;
+using DecisionsFramework.ServiceLayer;
+using DecisionsFramework.ServiceLayer.Services.ContextData;
+using Newtonsoft.Json;
+namespace Decisions.Equifax
+{
+    public class EquifaxUtilities
+    {
+        public static readonly Log log = new Log(EquifaxConstants.LogCat);
+
+        internal static ConsumerCreditReportResponse ExecuteCreditReportRequest(ConsumerCreditReportRequest request, string scope, string requestUrl)
+        {
+            string responseString = RequestSerializer(request, scope, requestUrl);
+
+            ConsumerCreditReportResponse limitedCreditResponse = new ConsumerCreditReportResponse();
+           
+                limitedCreditResponse = JsonConvert.DeserializeObject<ConsumerCreditReportResponse>(
+                    responseString, new JsonSerializerSettings {NullValueHandling = NullValueHandling.Ignore});
+                return limitedCreditResponse;
+        }
+        internal static PreQualificationOfOneResponse ExecutePrequalificationRequest(ConsumerCreditReportRequest request, string scope, string requestUrl)
+        {
+            string responseString = RequestSerializer(request, scope, requestUrl);
+            PreQualificationOfOneResponse preQualResponse = new PreQualificationOfOneResponse();
+            preQualResponse  = JsonConvert.DeserializeObject<PreQualificationOfOneResponse>(
+                responseString, new JsonSerializerSettings {NullValueHandling = NullValueHandling.Ignore});
+            
+            return preQualResponse;
+        }
+
+        private static string  RequestSerializer(ConsumerCreditReportRequest request, string scope, string requestUrl)
+        {
+            JsonSerializerSettings jsonSettings = new JsonSerializerSettings
+            {
+                NullValueHandling = NullValueHandling.Ignore
+            };
+
+            if (string.IsNullOrWhiteSpace(scope))
+                throw new LoggedException(EquifaxConstants.SETTINGS_CONFIGURATION_EXCEPTION);
+            string requestToken = GetOAuthToken(scope);
+            string requestString = JsonConvert.SerializeObject(request, jsonSettings);
+            var req = (HttpWebRequest) WebRequest.Create(requestUrl);
+            req.Headers.Add("Authorization: Bearer " + requestToken);
+            req.Method = "POST";
+            req.ContentType = "application/json";
+            req.ContentLength = requestString.Length;
+            req.AutomaticDecompression = DecompressionMethods.GZip;
+
+            Log log = new Log(EquifaxConstants.LogCat);
```

Comment:
This line can be removed. The class static logger will be used instead, without any other changes.